pkg/kubelet/prober/worker.go

```diff
@@ -56,6 +56,8 @@ type worker struct {
 	resultsManager results.Manager
 	probeManager   *manager
 
+	dependentManager results.Manager
```

Comment:
Only because there was no guarantee you passed in a readiness manager.  You could have others.
I will rename it.